### ISSUE TITLE
jinja include in template

### ISSUE THREAD

**Issue by ascii-ts** [convergeone/freeztp#45]
i'm trying to include several files into the config.
we want to use freeZTP in a larger scale across a lot sites.
Due to the histroric structure of the company almost every side uses different vlan setup and ise and so on.

so the basic idea is to 1 base file and than start including site specifics depending on the keystore value.

```
ztp set keystore kretest mgmt_vlan '410'
ztp set keystore kretest mgmt_ip 10.x.y.123
ztp set keystore kretest mgmt_sub 255.255.255.0
ztp set keystore kretest hostname kre-merged
ztp set keystore kretest location kre
ztp set keystore kretest ports '8'
ztp set idarray kretest FOC19xyz
ztp set association id kretest template basic
ztp set external-template basic file '/etc/ztp/templates/basic'
```
the end of my basic looks like this
```
{% include '/etc/ztp/templates/acl' %}
{% include '/etc/ztp/templates/{{ location }}/vlan' %}
{% include '/etc/ztp/templates/{{ location }}/ports_{{ ports }}' %}
{% include '/etc/ztp/templates/{{ location }}/acs' %}
{% include '/etc/ztp/templates/{{ location }}/misc' %}
{% include "/etc/ztp/templates/{{ location }}/upgrade" ignore missing %}
```
so the idea is to have a folder for each location here it is kre and have the site specific config in it.
When i try to merge config it fails
```
root at dekre-freeZTP in /etc/ztp/templates
$ ztp request merge-test kretest 
2019-05-24 08:33:03:   cfact.get_keystore_id: Checking Keystores and IDArrays for (kretest)
2019-05-24 08:33:03:   cfact.get_keystore_id: ID (kretest) resolved directly to a keystore
2019-05-24 08:33:03:   cfact.get_template: Looking up association for identity (kretest)
2019-05-24 08:33:03:   cfact.get_template: Found associated template (basic)
2019-05-24 08:33:03:   cfact.get_template: Template (basic) exists as an external-template. Returning
2019-05-24 08:33:03:   cfact.pull_keystore_values: Inserting IDArray keys
2019-05-24 08:33:03:   cfact.merge_final_config: Merging with values:
{
    "idarray_1": "FOC19xyz", 
    "mgmt_vlan": 410, 
    "location": "kre", 
    "mgmt_ip": "10.x.y.123", 
    "mgmt_sub": "255.255.255.0", 
    "hostname": "kre-merged", 
    "ports": 8, 
    "idarray": [
        "FOC19xyz"
    ]
}
##############################
Traceback (most recent call last):
  File "/bin/ztp", line 3767, in <module>
    interpreter()
  File "/bin/ztp", line 3634, in interpreter
    cfact.merge_test(sys.argv[3], "final")
  File "/bin/ztp", line 646, in merge_test
    console(j2template.render(kvalues))
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "<template>", line 69, in top-level template code
TypeError: no loader for this environment specified
root at dekre-freeZTP in /etc/ztp/templates
```

**Comment by PackeTsar** [comment]
This is a known issue and I have fixed it in an internal dev version. I will get it published out today

**Comment by PackeTsar** [comment]
This should be resolved now in v1.2.0
